hooks4git/scripts.py

```diff
@@ -99,45 +106,35 @@ def add_hooks(path=os.environ["PWD"]):
-                files_to_copy = system('ls', os.path.join(setup_path, 'git/hooks'))
-                for file in files_to_copy[1].split('\n'):
-                    if file not in ['__pycache__', '', 'hooks4git.py']:
-                        src = os.path.join(setup_path, 'git/hooks', file)
-                        target = os.path.join(git_path, 'hooks', file)
-                        copy(src, target)
-                print("Wow! hooks4git files were installed successfully! Thanks for hooking!")
-                print("If you are a courious person, take a look at .git/hooks folder.")
-                print("TIP: To get rid of the hooks, delete the .hooks4git.ini from your project.")
-            else:
-                if not standalone_run:
-                    message = '*****************************************************************\n'
-                    message += '* hooks4git is installed. Just run "hooks4git" to install the hooks.'
-                    print(message)
-                else:
-                    print("I am afraid I can't to that. Looks like your .git folder is not standard.")
-        else:
-            print("I am afraid I can't to that. You are not inside a GIT repo. Reach one and re-run this tool.")
+        path = os.path.abspath(path)
+        setup_path = os.path.abspath(setup_path)
+        hooks_path = get_hooks_path(git_path, "hooks", create_if_missing=True)
+        if hooks_path:
+            origin_config = os.path.join(setup_path, '.hooks4git.ini')
+            target_config = os.path.join(path, '.hooks4git.ini')
+            if os.path.isfile(target_config):
+                target_config = target_config.replace('.ini', '-' + __version__ + '.ini')
+            copy(origin_config, target_config)
+            files_to_copy = system('ls', os.path.join(setup_path, 'git/hooks'))
+            for file in files_to_copy[1].split('\n'):
+                if file not in ['__pycache__', '', 'hooks4git.py']:
+                    src = os.path.join(setup_path, 'git/hooks', file)
+                    target = os.path.join(git_path, 'hooks', file)
+                    copy(src, target)
+            print("Wow! hooks4git files were installed successfully! Thanks for hooking!")
+            print("If you are a courious person, take a look at .git/hooks folder.")
+            print("TIP: To get rid of hooks, comment lines on the .hooks4git.ini file.")
+        # else:
+        #     if not standalone_run:
+        #         message = '*****************************************************************\n'
+        #         message += '* hooks4git is installed. Just run "hooks4git" to install the hooks.'
+        #         print(message)
+        #     else:
+        #         print("I am afraid I can't to that. Looks like your .git folder is not standard.")
         return True
 
 
-class Post_install(install):
-    def run(self):
-        install.run(self)
-        Exec.add_hooks()
+# class Post_install(install):
```

Comment:
you are using a VCS, don't comment, remove the code. If need be you can find it using git

Reply:
I know... bad habit. This one, in particular, can be removed.